service/src/test/java/bio/terra/cbas/service/TestRunSetsService.java

```diff
@@ -43,29 +57,33 @@
 import org.databiosphere.workspacedata.model.RecordAttributes;
 import org.databiosphere.workspacedata.model.RecordResponse;
 import org.junit.jupiter.api.Test;
-import org.junit.jupiter.api.extension.ExtendWith;
-import org.mockito.junit.jupiter.MockitoExtension;
-import org.springframework.test.context.ContextConfiguration;
-
-@ExtendWith(MockitoExtension.class)
-@ContextConfiguration(
-    classes = {
-      CbasApiConfiguration.class,
-      GlobalExceptionHandler.class,
-      AsyncExceptionHandler.class
-    })
-class TestRunSetHelper {
+
+class TestRunSetsService {
 
   private final RunDao runDao = mock(RunDao.class);
   private final RunSetDao runSetDao = mock(RunSetDao.class);
+  private final MethodDao methodDao = mock(MethodDao.class);
+  private final MethodVersionDao methodVersionDao = mock(MethodVersionDao.class);
   private final CromwellService cromwellService = mock(CromwellService.class);
   private final WdsService wdsService = mock(WdsService.class);
   private final CbasApiConfiguration cbasApiConfiguration = mock(CbasApiConfiguration.class);
   private final UuidSource uuidSource = mock(UuidSource.class);
-
-  private final RunSetsHelper mockRunSetHelper =
-      new RunSetsHelper(
-          runDao, runSetDao, cromwellService, wdsService, cbasApiConfiguration, uuidSource);
+  private final ObjectMapper objectMapper = mock(ObjectMapper.class);
+  private final CbasContextConfiguration cbasContextConfiguration =
+      mock(CbasContextConfiguration.class);
+
+  private final RunSetsService mockRunSetsService =
```

Comment:
It's probably better to have a BeforeEach method to recreate this (and all the mocks) before each test, so there's no chance of leaking between them